client/admin/components/users/index.vue

```diff
@@ -78,6 +79,7 @@ const defaultPage = () => ({ sortBy: 'updatedAt', descending: true, page: 1 });
 
 export default {
   name: 'user-list',
+  mixins: [withKeys({ multiSelect: keys.shift })],
```

Comment:
Creates `multiSelect` computed that gets _activated_ with <kbd>shift</kbd> keydown event and _deactivated_ with keyup.